src/Examples/Flowsheets/ngcc/soec.py

```diff
@@ -214,56 +202,28 @@ def _add_units(self):
                 "e^-": -1,
             }
 
-        self.soec = SolidOxideCell(
-            default={
-                "has_holdup": True,
-                "control_volume_zfaces": zfaces,
-                "control_volume_xfaces_fuel_electrode": xfaces_electrode,
-                "control_volume_xfaces_oxygen_electrode": xfaces_electrode,
-                "control_volume_xfaces_electrolyte": xfaces_electrolyte,
-                "fuel_component_list": fuel_comps,
-                "fuel_triple_phase_boundary_stoich_dict": fuel_stoich_dict,
-                "inert_fuel_species_triple_phase_boundary": [],
-                "oxygen_component_list": oxygen_comps,
-                "oxygen_triple_phase_boundary_stoich_dict": oxygen_stoich_dict,
-                "inert_oxygen_species_triple_phase_boundary": ["N2", "Ar", "CO2", "H2O"],
-                "include_temperature_x_thermo": True,
-                "include_contact_resistance":True,
-            }
-        )
-        self.number_cells = pyo.Var(initialize=60e6, units=pyo.units.dimensionless)
-
-        self.fuel_inlet_translator = gum.Translator(
-            default={
-                "inlet_property_package": self.h2_side_prop_params,
-                "outlet_property_package": self.h2_side_prop_params,
-                "outlet_state_defined": True,
-            }
-        )
-
-        self.fuel_outlet_translator = gum.Translator(
-            default={
-                "inlet_property_package": self.h2_side_prop_params,
-                "outlet_property_package": self.h2_side_prop_params,
-                "outlet_state_defined": True,
-            }
-        )
-
-        self.oxygen_inlet_translator = gum.Translator(
+        self.soec_stack = soc.SolidOxideModuleSimple(
```

Comment:
To be really pedantic, we should probably use `soec_module` instead of `soec_stack`, because one module can contain many stacks.